topics/admin/tutorials/ansible-galaxy/tutorial.md

```diff
@@ -1859,6 +1859,118 @@ Now that your production-ready Galaxy is running, try registering a new user and
 
 In order to be the administrator user, you will need to register an account with the same email address you used in the group variables under the `admin_users` setting.
 
+## Job Configuration
+
+One of the most important configuration files for a large Galaxy server is the `job_conf.xml` file. This file tells Galaxy where to run all of the jobs that users execute. If Galaxy can't find a job conf file or none have been specified in the `galaxy.yml` file, it will use a default configuration. The default configuration is the `job_conf.xml.sample_basic` file. This file is located at: `/srv/galaxy/server/lib/galaxy/config/sample/job_conf.xml.sample_basic`, though there is a symlink to the file in `/srv/galaxy/server/config`.
+
+The job configuration file allows Galaxy to run jobs in multiple locations using a variety of different mechanisms. Some of these mechanisms include:
+* Local - Galaxy runs jobs on the same computer that Galaxy itself is running on.
+* DRMAA - Galaxy can connect to a cluster and submit jobs via a distributed resource manager such as Slurm, Condor, PBS Torque or Sun Grid Engine.
+* Pulsar - Galaxy can also send jobs to remote compute hosts over the internet using Pulsar.
+
+### The job conf file - basics
+
+The `job_conf.xml` has three basic sections:
+* **Plugins** - This section lists the types of resources that this Galaxy server is configured to use and tells Galaxy to load the drivers for each type.
+* **Destinations** - This section lists the different locations or queues etc that Galaxy can send jobs to. Each one has a name and uses a *plugin* to communicate with that location. They can specify things like the number of cpu's, amount of RAM to be allocated etc for DRMAA locations. Usually one of the destinations is set to be the default.
+* **Tools** - This section lists the various tools that Galaxy would like to send to the non-default *destination*. Each line in this section pairs up a tool in Galaxy with a particular job *destination*. Then everytime Galaxy gets a job for that particular tool, it is always sent to that *destination*.
+
+The basic `job_conf.xml` file looks like this:
+
+{% raw %}
+```xml
+<job_conf>
+    <plugins workers="4">
+        <plugin id="local" type="runner" load="galaxy.jobs.runners.local:LocalJobRunner"/>
+    </plugins>
+    <destinations>
+        <destination id="local" runner="local"/>
```

Comment:
In Barcelona, as we added to the `job_conf.xml` there was some confusion over the distinction between destination IDs and runner IDs. This is copied straight from `job_conf.xml.sample_basic` but I wonder if we should change this to:

```xml
        <plugin id="local_plugin" type="runner" load="galaxy.jobs.runners.local:LocalJobRunner"/>
```

and:

```xml
        <destination id="local_destination" runner="local_plugin"/>
```